resources/services/elasticbeanstalk/environments.go

```diff
@@ -450,6 +458,17 @@ func fetchElasticbeanstalkEnvironmentLinks(ctx context.Context, meta schema.Clie
 	return nil
 }
 
+// It takes a few minutes for an environment to be terminated
+// This ensures we don't error while trying to fetch related resources for a terminated environment
+func IsInvalidParameterValueError(err error) bool {
```

Comment:
maybe move this into a more generic place i.e client, as I am sure more resources might want to use this method.

Reply:
Done in https://github.com/cloudquery/cq-provider-aws/pull/751/commits/e17f2bf671a0d43a2bdccca2b6a3b828b83a40e1